source/features/update-pr-from-base-branch.tsx

```diff
@@ -51,9 +51,27 @@ async function handler({delegateTarget}: delegate.Event): Promise<void> {
 
 async function addButton(position: Element): Promise<void> {
 	const {base, head} = getBranches();
-	const {status} = await api.v3(`compare/${base}...${head}`);
+	const [comparison, deepData] = await Promise.all([
+		// TODO: Find how to determine whether the branch needs to be updated via v4
+		api.v3(`compare/${base}...${head}`),
 
-	if (status === 'diverged') {
+		// Some fields are just for testing
+		api.v4(`
+			repository() {
+				pullRequest(number: ${getConversationNumber()!}) {
+					baseRefOid
+					mergeable
+					viewerCanEditFiles
+					canBeRebased
+					mergeStateStatus
```

Comment:
I may be misunderstanding these [2 new fields](https://docs.github.com/en/graphql/overview/schema-previews#merge-info-preview), but they seem to always match. 
- Either it's conflicting, cannot be rebased and dirty
- or mergeable, can be rebased and clean

<img width="194" alt="Screen Shot 10" src="https://user-images.githubusercontent.com/1402241/177037246-58ba5d59-baa8-4adc-b14d-3e69da34d2c3.png">

<img width="193" alt="Screen Shot 9" src="https://user-images.githubusercontent.com/1402241/177037244-8441ae70-adda-435f-b58d-c74e85d3078d.png">